src/js/shot-generator/Editor.js

```diff
@@ -218,10 +223,10 @@ const SceneManager = connect(
 
     let clock = useRef(new THREE.Clock())
 
-    let orthoCamera = useRef(new THREE.OrthographicCamera( -4, 4, 4, -4, 0, 1000 ))
+    let orthoCamera = useRef(new THREE.OrthographicCamera( -4, 4, 4, -4, 1, 10000 ))
 
     let cameraHelper = useRef(null)
```

Comment:
Should we remove all references to `cameraHelper` since we’re not using it for anything anymore?

Reply:
removed all references